python/rmm/_cuda/gpu.py

```diff
@@ -78,13 +79,13 @@ def runtimeGetVersion():
     The version is returned as (1000 major + 10 minor). For example,
     CUDA 9.2 would be represented by 9020.
 
-    This function automatically raises CUDARuntimeError with error message
-    and status code.
+    This calls numba.cuda.runtime.get_version() rather than cuda-python due to
+    current limitations in cuda-python.
     """
-    status, version = cudart.cudaRuntimeGetVersion()
-    if status != cudart.cudaError_t.cudaSuccess:
-        raise CUDARuntimeError(status)
-    return version
+    # TODO: Replace this with `cuda.cudart.cudaRuntimeGetVersion()` when the
```

Comment:
Maybe we should have an RMM issue about this?

Reply:
Done: https://github.com/rapidsai/rmm/issues/949

Reply:
Issue filed: https://github.com/rapidsai/rmm/issues/948

Reply:
Thanks! 😄